configurator/frontend/src/stores/destinations.ts

```diff
@@ -168,6 +171,14 @@ class DestinationsStore implements IDestinationsStore {
     return this._destinations;
   }
 
+  public get hiddenDestinations() {
```

Comment:
It is a good practice to add the new public methods to the class signature to make those methods type safe. Just add `hiddenDestinations: DestinationData[]` and `allDestinations: DestinatinonData[]` to the `IDestinationsStore`